examples/browserify.js

```diff
@@ -22,6 +22,9 @@ function draw(words) {
   d3.select("body").append("svg")
       .attr("width", layout.size()[0])
       .attr("height", layout.size()[1])
+      .attr('viewBox', '0 0 ' + layout.size()[0] + ' ' + layout.size()[1])
+      .style('width', '100%')
+      .style('height', 'auto')
```

Comment:
This is the standard way to make images responsive. If you're seeing unexpected behavior, please make an example showing the overflow issue you're describing.